app/src/test/java/com/x8bit/bitwarden/data/platform/repository/SettingsRepositoryTest.kt

```diff
@@ -1055,11 +1080,12 @@ class SettingsRepositoryTest {
                 AUTHENTICATION_SYNC_KEY,
                 fakeAuthDiskSource.getAuthenticatorSyncUnlockKey(USER_ID),
             )
+            fakeAuthDiskSource.authenticatorSyncSymmetricKey.contentEquals(symmetricKey)
             coVerify { vaultSdkSource.getUserEncryptionKey(USER_ID) }
         }
 
     @Test
-    fun `setting isAuthenticatorSyncEnabled to false should clear authenticator sync key`() =
+    fun `isAuthenticatorSyncEnabled set to false should clear authenticator sync key`() =
```

Comment:
Can we update this to show that this also doesn't delete the `authenticatorSyncSymmetricKey`?

Reply:
I like that idea!

187468076919f841d300fc5b4caa564ce8160cdb